src/Utils.js

```diff
@@ -1,3 +1,78 @@
 (function(){
   LifeGamification.utils = {};
+  LifeGamification.work = {};
+  LifeGamification.refreshTimer;
+
+  LifeGamification.utils.setStartTime = function () {
+    return new Promise((resolve, reject) => {
+      LifeGamification.repository.getWork()
+      .then(function(result){
+        if(result){
+          LifeGamification.work = result;
+          resolve();
+        }
+      });
+    });
+  }
+
+  LifeGamification.utils.startTiming = function () {
+    if(!($('.timer__select-skill').val())){
+      console.log("Error: Add a skill first.");
+      return;
+    }
+    let currentTime = new Date().getTime();
+    let skillName = $('.timer__select-skill').val();
+    LifeGamification.repository
+      .setWork({startTime: currentTime, name: skillName})
+    .then(LifeGamification.utils.handleTimer());
+  }
+
+  LifeGamification.utils.endTiming = function () {
+    const skill = LifeGamification.
+      skillsCollection[LifeGamification.work.name];
```

Comment:
The name `name` is confusing in this line. I'm not really sure if it is work name or skill name.